styles.css

```
@@ -4781,18 +4781,16 @@ input:checked + .slider::before {
     box-shadow: 0 2px 4px rgb(0, 0, 0, 0.3);
 }
 
-.volume-bar,
-.fs-volume-bar {
+.volume-bar, .fs-volume-bar {
```

Stylelint (17.14.0)

[error] Expected selector ".volume-bar" to come before selector ".volume-controls .volume-bar", at line 3910 (no-descending-specificity)

(no-descending-specificity)